### PR TITLE
adding DeepJet pass to HLT-BTV validation plots

### PR DESCRIPTION
**Pull request by MaryShooshtari**
expected changes : new plots for the DeepJet BTagger in HLT-BTV validation
no external dependencies
 
Passes the basic test procedure suggested in the [CMSSW PR instructions](https://cms-sw.github.io/PRWorkflow.html)

### CODE REVIEW

**Comment by emanueleusai** [comment]
please test

**Comment by emanueleusai** [comment]
You PR introduces a crash:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-dec7e7/31927/runTheMatrix-results/11634.914_TTbar_14TeV+2021_DDDDB/step3_TTbar_14TeV+2021_DDDDB.log

See in particular these lines:
cmsRun: /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_13_1_X_2023-04-11-2300/src/HLTriggerOffline/Btag/src/HLTBTagPerformanceAnalyzer.cc:304: virtual void HLTBTagPerformanceAnalyzer::bookHistograms(dqm::legacy::DQMStore::IBooker&, const edm::Run&, const edm::EventSetup&): Assertion `hltPathNames_.size() == JetTagCollection_.size()' failed.

File: HLTriggerOffline/Btag/python/HltBtagValidation_cff.py [hunk 1]
```diff
@@ -47,9 +49,12 @@
 	'HLT_PFHT330PT30_QuadPFJet_75_60_45_40_v',
 	'HLT_PFHT400_SixPFJet32_DoublePFBTagDeepCSV_',
 	'HLT_PFHT400_SixPFJet32_DoublePFBTagDeepCSV_',
+	'HLT_PFHT400_SixPFJet32_DoublePFBTagDeepJet_', 
+	'HLT_PFHT400_SixPFJet32_DoublePFBTagDeepJet_',
```
Comment (perrotta) [inline HLTriggerOffline/Btag/python/HltBtagValidation_cff.py:53]: This line is a repetition of the previous one

**Comment by emanueleusai** [comment]
please test

**Comment by emanueleusai** [comment]
+1

**Comment by perrotta** [comment]
+1

**Pull request merged**